training-export-service/export.py

```
@@ -0,0 +1,227 @@
+                print(f"  Render error page {page_num}: {e}")
+                continue
+
+            # Get PDF page dimensions for normalisation
+            try:
+                import pikepdf
+                with pikepdf.open(pdf_path) as pdf:
+                    page = pdf.pages[page_num - 1]
+                    mb = page.MediaBox
+                    pdf_w = float(mb[2]) - float(mb[0])
+                    pdf_h = float(mb[3]) - float(mb[1])
+            except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)